packages/smart-adserver/src/components/smart-ad.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import { css, connect } from "frontity";
+import SmartAdserver, { SmartAdProps } from "../../types";
+import { Connect } from "frontity/types";
+
+/**
+ * Create styles for the iframe ad.
+ *
+ * @param callType - Type of call: "iframe" | "std".
+ * @param width - Width in px.
+ * @param height - Height in px.
+ *
+ * @returns The styles.
+ */
+const styles = (callType: string, width: number, height: number) =>
+  callType === "iframe"
+    ? css`
+        width: ${width}px;
+        height: ${height}px;
+      `
+    : css``;
```

Comment:
Can't we also style the div of `std`?

Ads are always a mess for Cumulative Layout Shift (https://web.dev/cls/) so I remember that in the past we always tried to add a `height`.

Maybe the `height` value can be `height` for the iframe and `min-height` for the `div`, so if a bigger ad is loaded it's not cropped.

@darerodz, what do you think?

Reply:
I agree. :+1:
I think we did something similar in the past.